service/settings.toml

```diff
@@ -5,3 +5,5 @@ DATABASE_URL = "sqlite:///:memory:"
 SEARCH_QUERY= 'topic:"flask-extension flask-extensions"'
 DELAY = 10
 PRIORITY = 100
+LOGGER_FILE = 'log.log'
+GITHUB_TOKEN = '5edc6e893d2c881d5b033f0fd9bfb1a54694727b'
```

Comment:
Este token é válido? se for seria bom você invalidar ele no seu github e então deixamos ele só de exemplo aqui.

Reply:
Perfeito, já solicitei uma nova versão.

Reply:
O CI se encarrega do token:
https://github.com/flask-extensions/flaskextensions.com/blob/master/.github/workflows/ci.yml#L60-L64